rotkehlchen/globaldb/cache.py

```diff
@@ -145,16 +140,27 @@ def globaldb_delete_general_cache(
         key_parts: Iterable[Union[str, GeneralCacheType]],
         value: Optional[str] = None,
 ) -> None:
-    """Deletes from cache in globaldb. If value is None deletes all entries with the provided
-    cache key. Otherwise, deletes only single entry `key - value`."""
-    cache_key = compute_cache_key(key_parts)
+    """Deletes from general_cache in globaldb.
+    if the value is None deletes all entries with the provided cache key.
+    Otherwise, deletes only single entry `key - value`."""
+    cache_key, _ = compute_cache_key(key_parts)
     if value is None:
-        write_cursor.execute('DELETE FROM general_cache WHERE key=?', (cache_key,))
+        write_cursor.execute(f'DELETE FROM {GENERAL_CACHE_TABLE_NAME} WHERE key=?', (cache_key,))
     else:
-        write_cursor.execute('DELETE FROM general_cache WHERE key=? AND value=?', (cache_key, value))  # noqa: E501
+        write_cursor.execute(f'DELETE FROM {GENERAL_CACHE_TABLE_NAME} WHERE key=? AND value=?', (cache_key, value))  # noqa: E501
 
 
-def globaldb_delete_general_cache_like(
+def globaldb_delete_unique_cache(
```

Comment:
As I said in the previous PR review this is not needed. Remove this function. You only use it in tests afaics. So that means it's not needed.

Reply:
even the one to delete general cache has no real usage now other than test. should i remove it too?

Reply:
yes I would say so

Reply:
done and rebased aswell. hopefully tests pass